ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -89,8 +89,8 @@ node('caasp-team-private-integration') {
            }
            if (pr_repo_label != null) {
                def branch_name = pr_repo_label.name.split(":")[1]
-               branch_repo = "http://download.suse.de/ibs/Devel:/CaaSP:/4.0:/Branches:/${branch_name}/SLE_15_SP1"
-               branch_registry = "registry.suse.de/devel/caasp/4.0/branches/${branch_name}/containers"
+               branch_repo = "http://download.suse.de/ibs/Devel:/CaaSP:/${repo_version}:/Branches:/${branch_name}/SLE_15_SP1"
+               branch_registry = "registry.suse.de/Devel/CaaSP/${repo_version}/Branches/${branch_name}/containers"
```

Comment:
mmm not sure why there is this change here... registry references with capital letters are not allowed... which makes me note that `${branch_name}` should turned in to lowercase characters on this string composition. In any case this is fix for master not the backport itself.

Reply:
wtf.

So how is this working on master then? is it just ignoring it? Are they not allowed or simply ignored?

Who the fuck thought that not allowing capital letters was a good idea???? Is this the 80's with their short-sighted tech ideas????

Reply:
> Who the fuck thought that not allowing capital letters was a good idea???? Is this the 80's with their short-sighted tech ideas????

I guess it was the same guy that decided to use some sort of URI for image references but without following regular URI separators and syntax.

> So how is this working on master then? is it just ignoring it? Are they not allowed or simply ignored?

On master it is already without capital letters, I guess there was a post fix for that later on that we might have missed to backport.

Reply:
found the fix, backported, good catch @davidcassany !